static/app/utils/profiling/canvasView.tsx

```diff
@@ -128,6 +148,32 @@ export class CanvasView<T extends {configSpace: Rect}> {
     return fromConfigView;
   }
 
+  fromTransformedConfigView(space: Rect): mat3 {
+    const fromConfigView = transformMatrixBetweenRect(
+      this.configView.transformRect(this.configSpaceTransform),
+      space
+    );
+
+    if (this.inverted) {
+      mat3.multiply(fromConfigView, space.invertYTransform(), fromConfigView);
+    }
+
+    return fromConfigView;
+  }
+
+  fromTransformedConfigSpace(space: Rect): mat3 {
+    const fromConfigView = transformMatrixBetweenRect(
+      this.configSpace.transformRect(this.configSpaceTransform),
+      space
+    );
+
+    if (this.inverted) {
+      mat3.multiply(fromConfigView, space.invertYTransform(), fromConfigView);
+    }
+
+    return fromConfigView;
+  }
```

Comment:
Is there a reason you opted to introduce new methods for the transformed variants? If we set `this.configSpaceTransform` to the identity matrix, it is equivalent to the non transformed variants.

Plus I see that you changed all instances of the `fromConfigView` to `fromTransformedConfigVIew` and same for all the other methods it seems. So why not just keep `fromConfigView`, embed the transformation into that directly, and set it to the identity matrix when we do not want a transform?

Reply:
as discussed on slack, this is because the grid renderer needs the non offset transform while the flamegraph needs the offset transform